### DISCUSSION TITLE
Backup your Jellyfin and don't update to 10.9

### DISCUSSION THREAD

**Discussion by MilosHavranek** [jellyfin/jellyfin#12506] [General]
10.9 is a sad story of bad luck for me, as after update from 10.8 to 10.9.9 some videos have out of sync audio or subtitles and DLNA is not working anymore. These things happen one way or another.
I had a recent backup that for some reason had to be deleted and now I cannot rollback to 10.8.
Don't make the same mistake, backup your Jellyfin while you can.

IDEA: Why does not jellyfin copy the database to a backup folder before migration? This might be a good and easy feature to implement. It would allow rollbacks in some cases

**Comment by sjorge** [comment]
DLNA moved to a plugin in 10.9, so make sure that one is installed.

There might actually be a library.db.bak# as I have a bunch of those. So you might be able to use one of those to rollback.

It's general good practice to make a backup before any major or minor version bumps.

**Reply by MilosHavranek** [reply]
I have that plugin, unfortunately it doesn't work (already reported). Thanks for that bak file, I'm gonna try it.

**Reply by DavidTunstall** [reply]
Maybe I'm in the wrong location, if you guys are all developers, but I have a really basic question.
I have Jellyfin 10.9.11 installed on Truenas scale.
I cant for the life of me figure how to get the DLNA function to work. I see loads of comments about DLNA functionality being broken in various releases. I have the network setting set t ohost network to advertise services, but I don't see any DLNA server located on the Jellfin server.
Also, I find the documentation of how to install a plugin (if there is such needed in this setup). How do I ACTUALLY get the plugin, and how do i install it? replies using small words and short sentences are probably within my capabilites!

**Reply by MilosHavranek** [reply]
There are plugins in the administration, there you can install it by clicking.
I managed it to somehow get working... Client Discovery Interval: 20sec; Blast Alive Messages: 30 sec; Send only to matched host UNCHECKED; Default User: NONE

**Reply by DavidTunstall** [reply]
Thanks for the reply, but 'in the administration' means where, exactly? Administration menus for Jellyfin via the Web Portal, or in Turenas admin menus themselves? Even Jellyfins documentation hub seems vague on exactly how to enable this. If I go to the Jellyfin web portal, there are options under the menu (three horizontal lines), and I see User/Settings but no obvious place to add plugins.
On the website https://github.com/jellyfin/jellyfin-plugin-dlna, there are a lot of entries, but not clear on what to download, or install.
Also I see people mention changing blastalive settings to help.
Again, where are these settings located? Sorry for dumb questions, but only just venturing into the Truenas/apps/github world.
